src/modules/map/components/openlayers/OpenLayersWMSLayer.vue

```diff
@@ -97,6 +98,9 @@ if (gutter.value !== -1) {
 // That means that data will not be requested if the map viewport is outside the extent.
 if (wmsLayerConfig.value.extent) {
     layer.setExtent(flattenExtent(wmsLayerConfig.value.extent))
+} else if (wmsLayerConfig.value instanceof GeoAdminWMSLayer) {
+    // do not request stuff outside our technical extent with our own layers.
+    layer.setExtent(LV95.getBoundsAs(projection.value).flatten)
 }
```

Comment:
Is this not dangerous, what if in future we add the `extent` to the GeoAdminWMSLayer, then this hack will not used anymore, maybe putting this `if` first is more future proof.

```javascript
if (wmsLayerConfig.value instanceof GeoAdminWMSLayer) {
    // do not request stuff outside our technical extent with our own layers.
    layer.setExtent(LV95.getBoundsAs(projection.value).flatten)
} else if (wmsLayerConfig.value.extent) {
    layer.setExtent(flattenExtent(wmsLayerConfig.value.extent))
}
```

Reply:
Wouldn't that be good that if the layer has some specific configured extent we use it instead of the "hack"?